BinaryEncoding.md

```diff
@@ -65,20 +65,40 @@ Yes:
 
 ## Global structure
 
-* A module contains:
-  * a header followed by
-  * a table (sorted by offset) containing, for each section, its type and offset (within the module), followed by
-  * a sequence of sections.
+* A module contains (in this order):
+  - A header, containing:
+    + The [magic number](https://en.wikipedia.org/wiki/Magic_number_%28programming%29)
+    + Other data TBD (possibly entrypoint, memory bitness, source language, etc.)
```

Comment:
What is memory bitness?

At an initial glance, source language seems like something we'd specifically try to avoid including in the main header, because it suggests special magical per-source-language semantics.